src/App.jsx

```diff
@@ -1,13 +1,54 @@
-
-import './App.css'
+import AddingProduct from "./compements/AddingProduct.jsx";
+import './styles/App.css';
+// eslint-disable-next-line no-unused-vars
+import React from "react";
+import Form from "./compements/Form.jsx";
 
 function App() {
+ let [strawberry,setStrawberry] = React.useState(0);
+ let [banana,setBanana] = React.useState(0);
+ let [apple,setApple] = React.useState(0);
+ let [kiwi,setKiwi] = React.useState(0);
+
+ const [firstname, setFirstname] = React.useState("");
+ const [lastname, setLastname] = React.useState("");
+ const [age, setAge] = React.useState(null);
+ const [postcode, setPostcode] = React.useState("");
+ const [deliveryType, setDeliveryType] = React.useState("");
+ const [selectedOption, setSelectedOption] = React.useState("");
+ const [message, setMessage] = React.useState("");
+ const [terms, setTerms] = React.useState(false);
+
 
-  return (
+
+ return (
     <>
         <h1>Fruitmand bezorgservice</h1>
+        <section>
+        <AddingProduct name={"🍓Aarbeien "} dataValue={strawberry} dataSet={setStrawberry} />
+        <AddingProduct name={"🍌Bananen "} dataValue={banana} dataSet={setBanana} />
+        <AddingProduct name={"🍎Appels "} dataValue={apple} dataSet={setApple} />
+        <AddingProduct name={"🥝Kiwi's "} dataValue={kiwi} dataSet={setKiwi} />
+        </section>
+        <br/>
```

Comment:
de <br/> is niet heel netjes om te gebruiken, hiervoor hebben we padding en margin :)